.github/workflows/validate_examples.yml

```
@@ -21,7 +21,7 @@ jobs:
           cache: "pip"
       - name: Install Python dependencies
         run: |
-          python3 -m pip install check-jsonschema==0.31.0 pyshacl==0.30.0 spdx3-validate==0.0.5
+          python3 -m pip install check-jsonschema==0.31.0 spdx3-validate==0.0.5
```

```suggestion
          python3 -m pip install check-jsonschema==0.34.1 spdx3-validate==0.0.5
```

Optionally, we can also update the check-jsonschema.


Or we can do this separately.